### PR TITLE
refactored CSS to SASS

### PR DESCRIPTION
**Pull request by nalnaji**
We should make a transition to start using SASS. It's more of an industry standard and our CSS is getting messy.

@jonathanshum  @DarryQueen

### CODE REVIEW

**Comment by DarryQueen** [comment]
Consider also organizing the files. Right now, there's no obvious place as to which attribute belongs in which file.

**Comment by DarryQueen** [comment]
Interesting... Not able to decrypt because you're on a different repository... And here I thought I had such an elegant solution, too...

File: app/assets/stylesheets/custom.css [hunk 1]
```diff
@@ -1,214 +1,3 @@
 html{
   min-width: 500px;
 }
```
Comment (DarryQueen) [inline app/assets/stylesheets/custom.css]: Why'd you decide to keep these lines?

Comment (nalnaji) [inline app/assets/stylesheets/custom.css]: I tried putting it in application.css where it belongs, but that broke it xD

Comment (DarryQueen) [inline app/assets/stylesheets/custom.css]: This is fine, then. Put a space after `html`.

File: app/assets/stylesheets/dashboard.sass [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  .goals
+    list-style-type: none
+    padding-left: 15px
+    padding-right: 15px
+    padding-top: 10px
+  ul li:last-child
+    border-bottom-left-radius: 15px !important
+    border-bottom-right-radius: 15px !important
+
+/* Dashboard Video Modules */
```
Comment (DarryQueen) [inline app/assets/stylesheets/dashboard.sass]: Comment inapt?

File: app/assets/stylesheets/videos.sass [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    padding-right: 0px
+
+  .col-md-6
+    padding-left: 10px
+    padding-right: 10px
+
+@media (min-width: 992px)
+  .video-filter
+    height: 40px
+
```
Comment (DarryQueen) [inline app/assets/stylesheets/videos.sass]: Unnecessary linebreak.

File: app/assets/stylesheets/navbar.sass [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    position: relative
+    margin-right: 5px
+  .dropdown-toggle
+    padding-right: 5px
+  .nav-mini
+    font-weight: bold
+  .navbar-right
+    margin-right: 0px
+
+/* For mobile viewers */
```
Comment (DarryQueen) [inline app/assets/stylesheets/navbar.sass]: Yo, keep it consistent with the punctuation at the end of comments.

File: app/assets/stylesheets/preview.sass [hunk 5]
```diff
@@ -0,0 +1,3 @@
+/* Links on preview page. */
```
Comment (DarryQueen) [inline app/assets/stylesheets/preview.sass]: While you're at that, add a linebreak after comments. Or get rid of them all, but just keep it consistent.

File: app/assets/stylesheets/social_media.sass [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+  &:hover, &:focus
+    color: #fff
+
+.btn-facebook:hover
+  background-color: #3b5998
+
+.btn-google-plus:hover
+  background-color: #dd4b39
+
+.no-padding
```
Comment (jonathanshum) [inline app/assets/stylesheets/social_media.sass]: .no-padding should probably go in some general elements file

File: app/assets/stylesheets/videos.sass [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+    width: 100%
+    height: auto
+  iframe
+    position: absolute
+    top: 0
+    left: 0
+    width: 100% 
+    height: 100%
+
+.module-padding
```
Comment (jonathanshum) [inline app/assets/stylesheets/videos.sass]: This one should be in a module panel specific sass file too.

File: app/assets/stylesheets/videos.sass [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+    padding-left: 0px
+
+  .padding-button:last-child
+    padding-right: 0px
+
+  .col-md-6
+    padding-left: 10px
+    padding-right: 10px
+
+@media (min-width: 992px)
```
Comment (jonathanshum) [inline app/assets/stylesheets/videos.sass]: Remove this

**Pull request closed without merging**